src/components/UNSTABLE__Shell/Shell.stories.js

```diff
@@ -0,0 +1,299 @@
+/**
+ * @file Shell stories.
+ * @copyright IBM Security 2020
+ */
+
+import Help20 from '@carbon/icons-react/lib/help/20';
+import Menu20 from '@carbon/icons-react/lib/menu/20';
+import Notification20 from '@carbon/icons-react/lib/notification/20';
+import User20 from '@carbon/icons-react/lib/user/20';
+import Settings20 from '@carbon/icons-react/lib/settings/20';
+
+import { action } from '@storybook/addon-actions';
+import { boolean, text } from '@storybook/addon-knobs';
+
+import React from 'react';
+
+import { disableCentered, meta, patterns } from '../../../.storybook';
+
+import {
+  Button,
+  HeaderAction,
+  HeaderActions,
+  IconButton,
+  Link,
+  Nav,
+  NavItem,
+  NavList,
+  NotificationsPopover,
+  PopoverAccount,
+  PopoverAccountItem,
+  PopoverAccountList,
+  PopoverFooter,
+  PopoverNotification,
+  ProfilePopover,
+  ShellHeader,
+  ShellHeaderName,
+  ShellReturnToBanner,
+  ShellSkipToContent,
+  ShellToolbar,
+  ToolbarAction,
+  UNSTABLE__Shell,
+} from '../..';
+
+const props = {
+  name: () => ({
+    children: text('`ShellHeaderName` children (`children`)', 'Application'),
+    brand: text('`ShellHeaderName` brand (`brand`)', 'Security'),
+    href: '#',
+    prefix: 'IBM',
+  }),
+
+  profile: () => ({
+    name: text('`ProfilePopover` name (`name`)', 'Sample'),
+    surname: text('`ProfilePopover` surname (`surname`)', 'User'),
+  }),
+
+  account: () => ({
+    children: text('`PopoverAccount` children (`children`)', 'Account 1'),
+    account: text('`PopoverAccount` account (`account`)', 'Account 1'),
+    id: text('`PopoverAccount` identifier (`id`)', 'Account 1'),
+  }),
+};
+
+const { account: accountProps, name: nameProps, profile: profileProps } = props;
+
+export const Default = () => (
+  <UNSTABLE__Shell>
+    <ShellHeader>
+      <ShellHeaderName {...nameProps()} />
+
+      <HeaderActions>
+        <HeaderAction>
+          <Button>Create an account</Button>
+        </HeaderAction>
+
+        <HeaderAction>
+          <Button kind="secondary">Sign in</Button>
+        </HeaderAction>
+      </HeaderActions>
+    </ShellHeader>
+  </UNSTABLE__Shell>
+);
+
+export const skipToContent = () => (
```

Comment:
For skip to content stories in particular, it is helpful to include content to skip to (i.e., not just a "main" target but also something inside, to demonstrate to users how tab order jumps to the skipped-to area)

Reply:
Good point! Done